client/components/recycleAndReuseComponents/Steps/StepThree/Geolocation/InfoBox.jsx

```diff
@@ -32,14 +32,21 @@ export const InfoBox = ({ content, handleCloseInfoBox, homeMarker, displayDirect
 					<Button
 						mt={1}
 						size="xs">
-							<Link href={`https://www.google.com/maps/dir/?api=1&origin=${homeMarker.current.lat}%2C${homeMarker.current.lng}&destination=${content.props.marker.postal}`} target="_blank">
-								Click for directions
-							</Link>
+							<RedirectToGoogleMap origin={homeMarker} dest={content.props.marker.postal} />
 					</Button>
 					:
 					""
 				}
 			</Box>
 		</div>
 	</Box>
+);
+
+const RedirectToGoogleMap = ({ origin, dest }) => (
```

Comment:
haha over kill but i like. maybe if other pages need it, can put this into a common folder - but that's a future problem